main.py

```diff
@@ -1204,6 +1456,11 @@ def test_4electrons_4orbital(self):
         integral_driven_indices = Test_MinimalEquivalence.simplify_indices(h.H_indices(psi, psi))
         self.assertListEqual(determinant_driven_indices, integral_driven_indices)
 
+        for (a, b), idx4, phase in integral_driven_indices:
```

Comment:
We should create a new tests class (Test_Category), and put all the `test utlity` function inside this class

Reply:
done